R/pkg/R/DataFrame.R

```diff
@@ -455,6 +455,17 @@ setMethod("createOrReplaceTempView",
               invisible(callJMethod(x@sdf, "createOrReplaceTempView", viewName))
           })
 
+#' @param tableName A character vector containing the name of the table
+#' @rdname createOrReplaceTempView
+#' @name registerTempTable
```

Comment:
looking at R practice on this, it seems we should have left registerTempTable.rd make to it registerTempTable-deprecated for clarity. we haven't done that at all for all the deprecation we have in 2.0.0 so we should probably scrub that shortly.

i'll make the change for this now.